src/test/java/org/gridsuite/study/server/VoltageInitTest.java

```diff
@@ -397,6 +413,18 @@ public void testVoltageInit() throws Exception {
 
         assertTrue(TestUtils.getRequestsDone(1, server).stream().anyMatch(r -> r.matches("/v1/results/" + VOLTAGE_INIT_RESULT_UUID + "/status")));
 
+        mockMvc.perform(
+                post("/v1/studies/{studyUuid}/voltage-init/parameters", studyNameUserIdUuid)
+                        .header("userId", "userId")
+                        .contentType(MediaType.ALL)
+                        .content(VOLTAGE_INIT_PARAMETERS_JSON)).andExpect(
+                status().isOk());
+
+        assertTrue(TestUtils.getRequestsDone(2, server).stream().allMatch(r -> r.matches("/v1/parameters/" + VOLTAGE_INIT_PARAMETERS_UUID) || r.matches("/v1/results/invalidate-status.*")));
+        //remove notif about study updating due to parameters changes
+        output.receive(1000);
```

Comment:
Perhaps just test the notification type for no regression ?

Reply:
I already did the test above in `testVoltageInitParameters()`

Reply:
This is what we agreed with the team ?

Reply:
No purge

Reply:
changed it